notebooks/aiida_datatypes_viewers.ipynb

```diff
@@ -88,7 +88,7 @@
    "source": [
     "StructureData = DataFactory('structure')\n",
     "s = StructureData(ase=m)\n",
-    "vwr = viewer(s.store(), downloadable=True)\n",
+    "vwr = viewer(s.store())\n",
```

Comment:
"downloadable" parameter does not make any difference for StructureViewerWidget.